# 문서 작성 규칙

## 구조
```
docs/
├── adr/        # 기술 결정 기록
├── specs/      # 도메인 스펙
└── rules/      # 컨벤션 및 규칙
```

## ADR
- 기술 주제별 파일 분리 (예: `architecture.md`, `http-client.md`)
- 한 파일 내에서 결정사항을 누적
- 항목: 날짜, 상태(채택/폐기), 컨텍스트, 결정

## Specs
- 도메인 단위 파일 분리
- 항목: 역할, 책임, 핵심 객체, 인바운드/아웃바운드

## Rules
- 주제별 파일 분리

## 문체
- 개조식, 키워드 중심으로 작성
- 설명문보다 구조와 핵심만 남김
- `→`로 인과/흐름 표현, 접속사 생략

```
// bad
Java 21의 가상 스레드가 정식 출시되면서, 동기 코드로도
비동기 수준의 처리량을 확보할 수 있게 되었다.

// good
Java 21 가상 스레드: 동기 코드로 비동기 수준 처리량 확보 가능
```
